src/datahike/index/hitchhiker_tree/upsert.cljc

```diff
@@ -26,7 +26,7 @@
   (-affects-key [_] key)
   (-apply-op-to-coll [_ kvs]
     (-> (or (remove-old kvs key (partial dissoc kvs)) kvs)
-      (assoc key value)))
+      (assoc key nil)))
```

Comment:
Can you remove the value also the UpsertOp record itself?

Reply:
Done.

Reply:
Ooppps, no, it is not done yet.

Reply:
Finally done now.